### PR TITLE
Renaming module part 2.

### LINKED ISSUE #12: Rename the module?
We've [renamed the GitHub repo](https://github.com/SciTools/cf_units/issues/7), but we've not renamed the Python package/module name.

### PR DESCRIPTION
**Pull request by ocefpaf**
Should Fix #12

### CODE REVIEW

**Comment by rhattersley** [comment]
Looks like this needs rebasing.

**Comment by ocefpaf** [comment]
@rhattersley not anymore :wink:

**Comment by rhattersley** [comment]
I did a quick grep for "iris" and "unit\>" and it threw up a couple of things:
1. `IRIS_EPOCH`
2. Lots of `import unit` and subsequent `unit.<method>` calls in the docstrings.

**Comment by ocefpaf** [comment]
Is `IRIS_EPOCH` -> `EPOCH` OK?

**Comment by rhattersley** [comment]
> Is IRIS_EPOCH -> EPOCH OK?

Yeah, seems OK.

File: cf_units/__init__.py [hunk 1]
```diff
@@ -17,4 +17,7 @@
 
 __version__ = '0.0.1'
 
-from .unit import *
+from .cf_units import *
+from .cf_units import (CALENDAR_STANDARD, CALENDARS, UT_NAMES, UT_DEFINITION,
```
Comment (ocefpaf) [inline cf_units/__init__.py:21]: @rhattersley should I those to `__all__` in `cf_units.py` or is this OK?

**Comment by ocefpaf** [comment]
@rhattersley  this time I tried to be very careful with the renaming and I believe we should be OK now.  I also ran `nosetests --verbose --with-doctest cf_units` to check the doctests and all but one passed:

``` python
File "cf_units.py", line 1961, in cf_units.cf_units.Unit.date2num
Failed example:
    u.date2num(datetime.datetime(1970, 1, 1, 5))
Expected:
    5.00000000372529
Got:
    5.0000000037252903
```

But I don't think that this precision is an issue.

**Comment by ocefpaf** [comment]
@rhattersley I want to start adding the tests (#2), but I want to avoid adding them and then re-naming everything.  So I am waiting for this one to start the test migration.

Do you think this PR is OK to go?

**Comment by rhattersley** [comment]
> Do you think this PR is OK to go?

It's close enough. :grin:

**Pull request merged**